packages/grafana-i18n/src/types/index.d.ts

```diff
@@ -0,0 +1 @@
+import './dates';
```

Comment:
Is this best practices to import like this into a `d.ts` file? Doesn't this turn this file into a module instead of a global declaration? Could we declare the functions directly in the file instead?